playwright/test/helpers/search.ts

```diff
@@ -85,6 +85,7 @@ export const clickImageSearchResultItem = async (
     .getByTestId('image-search-results-container')
     .locator(`ul:first-child > li:nth-child(${nthChild}) a`)
     .click();
+  await slowExpect(page.getByLabel('View expanded image')).toBeVisible();
```

Comment:
Moved the expect here to align with other helper functions that always "test themselves"!